hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -118,18 +124,18 @@
   /**
    * Hudi backed table metadata writer.
    *
-   * @param hadoopConf               - Hadoop configuration to use for the metadata writer
-   * @param writeConfig              - Writer config
-   * @param engineContext            - Engine context
-   * @param actionMetadata           - Optional action metadata to help decide bootstrap operations
-   * @param <T>                      - Action metadata types extending Avro generated SpecificRecordBase
+   * @param hadoopConf - Hadoop configuration to use for the metadata writer
+   * @param writeConfig - Writer config
+   * @param engineContext - Engine context
+   * @param actionMetadata - Optional action metadata to help decide bootstrap operations
+   * @param <T> - Action metadata types extending Avro generated SpecificRecordBase
    * @param inflightInstantTimestamp - Timestamp of any instant in progress
    */
   protected <T extends SpecificRecordBase> HoodieBackedTableMetadataWriter(Configuration hadoopConf,
-                                                                           HoodieWriteConfig writeConfig,
-                                                                           HoodieEngineContext engineContext,
-                                                                           Option<T> actionMetadata,
-                                                                           Option<String> inflightInstantTimestamp) {
+      HoodieWriteConfig writeConfig,
```

Comment:
nit: can we avoid changing formatting of these lines if there are no changes to these lines? Is this format on save doing hudi style changes?